component/src/main/java/org/wso2/carbon/identity/authenticator/office365/Office365Authenticator.java

```diff
@@ -42,29 +43,35 @@
 import org.wso2.carbon.identity.application.common.model.ClaimMapping;
 import org.wso2.carbon.identity.application.common.model.Property;
 import org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants;
-import org.wso2.carbon.identity.base.IdentityConstants;
-import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.identity.core.util.IdentityIOStreamUtils;
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
 
 /**
- * Authenticator of Outlook
+ * Authenticator of Office365.
+ *
+ * @since 1.0.1
```

Comment:
A new class was added in a patch release?

Reply:
Normally, if it is minor changes, we release tag like(1.0.2..).

Reply:
Sorry but that is for the patch release. The version strategy is Major.Minor.Patch or are we following a different one for wso2-extensions repos?

Reply:
For wso2-extensions, once we fix or do improvement, we release extension, AFAIK, we follow Major.Minor version strategy. If it is minor improvement or fix, we release minor version and if API changes, we release major version.